pkg/render/logstorage.go

```diff
@@ -359,14 +360,9 @@ func (es *elasticsearchComponent) podTemplate() corev1.PodTemplateSpec {
 	}
 
 	sc := securitycontext.NewRootContext(false)
```

Comment:
I think this securitycontext is for the main elasticsearch container. It should use `securitycontext.NewNonRootContext()`.

Reply:
Done